### PR TITLE
Assign remaining test/e2e/common test to SIGs

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #98326 

```release-note
NONE
```

/kind cleanup
/priority important-soon

### CODE REVIEW

File: test/e2e/common/downwardapi.go [hunk 1]
```diff
@@ -284,6 +284,7 @@ var _ = SIGNodeDescribe("Downward API", func() {
 	})
 })
 
+// TODO(#98326): Is this sig-node or sig-storage test?
```
Comment (wojtek-t) [inline test/e2e/common/downwardapi.go:287]: @dchen1107 @derekwaynecarr - can you please advise on it?

[it's also possible that split by SIG maybe at the level of individual tests here]

File: test/e2e/common/expansion.go [hunk 2]
```diff
@@ -30,6 +30,8 @@ import (
 // These tests exercise the Kubernetes expansion syntax $(VAR).
 // For more information, see:
 // https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/expansion.md
+//
+// TODO(#98326): Is this sig-node or sig-storage?
```
Comment (wojtek-t) [inline test/e2e/common/expansion.go:34]: @dchen1107 @derekwaynecarr - can you please advise on it?

[it's also possible that split by SIG maybe at the level of individual tests here]

**Comment by spiffxp** [comment]
/override pull-kubernetes-bazel-test
has been removed

**Comment by spiffxp** [comment]
/uncc @andrewsykim @cheftako 
/assign @ehashman @derekwaynecarr @SergeyKanzhelev 
I would like to hear from someone in SIG Node if these test assignments are ok

**Review by spiffxp — CHANGES_REQUESTED** [review]
Needs mechanical changes but I agree in principle with the assignments.  Would like to hear from @kubernetes/sig-node-pr-reviews though

File: test/e2e/common/container_probe.go [hunk 3]
```diff
@@ -45,7 +45,7 @@ const (
 	defaultObservationTimeout = time.Minute * 4
 )
 
-var _ = framework.KubeDescribe("Probing container", func() {
+var _ = SIGNodeDescribe("Probing container", func() {
```
[the remark this answers is not in the document]
Comment (wojtek-t) [reply test/e2e/common/container_probe.go:48]: I consciously used SIGNodeDescribe to make it easier to review for SIG-node folks.
I will do the second step of moving stuff around in a subsequent PR.

[the remark this answers is not in the document]
Comment (spiffxp) [reply test/e2e/common/container_probe.go:48]: OK, SGTM

**Review by spiffxp — APPROVED** [review]

File: test/e2e/common/downwardapi.go (hunk 1 above)
Comment (dchen1107) [reply test/e2e/common/downwardapi.go:287]: It is a joint project initiated by both SIG Storage and SIG Node, but driven by SIG storage later. Mark it for SIG Storage please.

File: test/e2e/common/expansion.go (hunk 2 above)
Comment (dchen1107) [reply test/e2e/common/expansion.go:34]: SIG Node please.

File: test/e2e/common/downwardapi.go (hunk 1 above)
Comment (wojtek-t) [reply test/e2e/common/downwardapi.go]: Thanks - done

File: test/e2e/common/expansion.go (hunk 2 above)
Comment (wojtek-t) [reply test/e2e/common/expansion.go]: thanks - done

**Comment by spiffxp** [comment]
/test pull-kubernetes-conformance-image-test 
(not required but I'm curious)

**Pull request merged**